src/renderer/components/kube-object-details/kube-object-details.tsx

```diff
@@ -62,7 +61,7 @@ export class KubeObjectDetails extends React.Component {
         .getStore(this.path)
         ?.getByPath(this.path);
     } catch (error) {
-      logger.error(`[KUBE-OBJECT-DETAILS]: failed to get store or object: ${error}`, { path: this.path });
+      console.error(`[KUBE-OBJECT-DETAILS]: failed to get store or object: ${error}`, { path: this.path });
```

Comment:
Hmm. Should not use explicit console.error instead of abstracted logger.

Reply:
Fair enough, I don't exactly remember why I did this since in #4317 I revert this back.... will revert.